### PR TITLE
Modular Imports to reduce size, fix warning#521

### LINKED ISSUE #521: Use Modular Import to Reduce Bundle Size
**Actual Behaviour**

Whole antd is being imported, which increases the bundle size

**Expected Behaviour**

Use Modular import plugin for babel compatible with antd.

**Screenshots of the issue**

screen shot 2018-09-21 at 10 02 01 pm

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #521 

Changes: Fixes warning, reduces size

Surge Deployment Link: https://pr-521-fossasia-susi-accounts.surge.sh

Updated link: mean-arch.surge.sh

### CODE REVIEW

File: src/components/Admin/Admin.js [hunk 1]
```diff
@@ -102,7 +102,7 @@ class Admin extends Component {
 
     return (
       <div>
-        {cookies.get('showAdmin') === 'true' ? (
+        {cookies.get('showAdmin') === 'false' ? (
```
Comment (Akshat-Jain) [inline src/components/Admin/Admin.js]: Why is this change done? Please explain. Rest PR looks good. Thanks! :-)

Comment (fragm3) [reply src/components/Admin/Admin.js]: @Akshat-Jain I was checking for Admin component(which requires access), if its working. Forgot to change it, thanks for pointing out.

**Comment by fragm3** [comment]
@anshumanv its probably surge bug showing page not found. Try running on localhost, it would work fine

**Comment by fragm3** [comment]
@anshumanv oh, I had done some changes, you can see a commit in the feeds. 
Updated link: [mean-arch.surge.sh](http://mean-arch.surge.sh)

**Comment by anshumanv** [comment]
You gave wrong PR link in the PR comment. :worried: 
It works.

**Pull request merged**